doc/release-notes-32604.md

```diff
@@ -0,0 +1,10 @@
+Logging
+-------
+Unconditional logging to disk is now rate limited by giving each source location
+a quota of 1MiB per hour. Unconditional logging is any logging with a log level
+higher than debug, that is `info`, `warning`, and `error`. All logs will be
```

Comment:
nit: Conditional logs (or rather, everything with `should_ratelimit=false`) don't get the prefix. Perhaps the better/easier UX is to just apply it to all logs?

```suggestion
higher than debug, that is `info`, `warning`, and `error`. All unconditional logs will be
```